v3-nginx/Dockerfile

```diff
@@ -10,6 +10,7 @@ ARG MODSEC_VERSION=3.0.6 \
 
 RUN set -eux; \
     apt-get update; \
+    unset LD_LIBRARY_PATH; \
```

Comment:
Can you add a comment here please?

Reply:
Finally, after removing it completely, the build succeeded. Sometimes, the upstream build breaks 🤷

Reply:
🙄